### ISSUE TITLE
Definition/distinction between Pityriasis streptogenes/alba ?

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#7144]
MONDO:0024345 Pityriasis streptogenes has no definition and I cannot find any articles in Pubmed using this string. Likewise a google search doesn't return any specific matches on this specific term. There do seem to be multiple hits connecting pityriasis rosea and streptococcus infection. (including https://www.ncbi.nlm.nih.gov/books/NBK448091/ )  

MONDO:0024349  Pityriasis alba also has no definition in Mondo, but I was able to find some clearer references to this condition.  i.e.  https://www.ncbi.nlm.nih.gov/books/NBK431061 .

UMLS has the two SNOMED CT codes on the same CUI (C0085657) , possible because another one of their sources "SNOMED International" term Pityriasis alba (D0-22163) includes 'pityriasis simplex' and 'pityriasis streptogenes' as synonyms. 

Bottom line ask is this: can Mondo find any additional definitions or information for these two conditions to help determine if they are, indeed, distinct conditions with specific clinical presentations? Or bring this to some dermatology expert for review?

**Comment by nicolevasilevsky** [comment]
@katiermullen could you work on this one? thx!

**Comment by katiermullen** [comment]
I am not a dermatology expert. I agree there no pubmed articles with the string "Pityriasis streptogenes." 

I would propose:

1. Definition for pityriasis rosea: A common acute, self-limited papulosquamous dermatosis. It is characterized by a herald patch followed by scaly oval patches on the trunk and proximal extremities in a "Christmas-tree" appearance. Source: PMID:32964824
3. Pityriasis streptogenes could be an obsoletion candidate with exclusion reason MONDO:excludeCausalAgents - previous strep infection is a suggested cause of pityriasis rosea, but that connection does not appear well established.
4. Definition for pityriasis alba: A prevalent and benign dermatological condition that predominantly affects children and adolescents. The name "pityriasis alba" derives from its appearance, where "pityriasis" denotes the fine scales and "alba" signifies the pale color (hypopigmentation). Source: https://www.ncbi.nlm.nih.gov/books/NBK431061/

**Comment by nicolevasilevsky** [comment]
**to do:** add MONDO:0024345 'pityriasis streptogenes' to obsoletion list with exclusion reason MONDO:excludeCausalAgents @sabrinatoro

**Comment by nicolevasilevsky** [comment]
I added a def to pityriasis alba

**Comment by twhetzel** [comment]
Checking in on what is left to do with this ticket:
- "pityriasis rosea" (MONDO:0006601) already has a definition with reference NCIT:P378:
```
A mild, self-limited skin disorder that is most commonly seen in children and young adults. It is characterized by an initial large round spot on the chest, abdomen, or back, often referred to as a herald patch, that is usually followed within a week by a distinctive pattern of similar but smaller papules on the torso, arms, and legs. There may also be itching, especially when overheated.
```
Katie in a [comment above](https://github.com/monarch-initiative/mondo/issues/7144#issuecomment-1954728153) proposed:
```
A common acute, self-limited papulosquamous dermatosis. It is characterized by a herald patch followed by scaly oval patches on the trunk and proximal extremities in a "Christmas-tree" appearance. Source: PMID:32964824
```
**Question**: Are any changes needed in the definition?

- "Pityriasis streptogenes" (MONDO:0024345) will be obsoleted as indicated in comments above

**Comment by katiermullen** [comment]
@twhetzel I think that all that is left to do is obsolete "Pityriasis streptogenes" (MONDO:0024345). Since "pityriasis rosea" (MONDO:0006601) already has a definition with reference NCIT:P378, you can ignore the definition that I proposed. 

Thank you!
